templates/shareaholic-settings.html.twig

```diff
@@ -0,0 +1,17 @@
+<div class='wrap'>
+  <script>
+  window.ShareaholicConfig = {
+    apiKey: "{{ apiKey }}",
+    verificationKey: "{{ verificationKey }}",
+    apiHost: "{{ apiHost }}",
+    serviceHost: "{{ serviceHost }}",
+    assetHost: "{{ assetHost }}",
+    assetFolders: true,
+    origin: "drupal",
+    language: "{{ language }}"
+  };
+  </script>
+  <div id="root" class="shr-site-settings"></div>
+  <script class="shr-app-loader__site-settings" src="https://cdn.shareaholic.net/ui-site-settings/loader.js"></script>
+  <script src="https://dsms0mj1bbhn4.cloudfront.net/assets/pub/loader-reachable.js" async></script>
+</div>
```

Comment:
Add chat support widget:

```
<script src="https://dsms0mj1bbhn4.cloudfront.net/assets/pub/shareaholic-chat.js" async></script>
```

Reply:
If we will add chat here, it will show only when react app will run. Do we want this or rather to have chat there always, regardless of any errors (which it could help to deal with)?

Reply:
Chat support should be shown on Shareaholic related settings pages always (App Manager and Advanced Settings), regardless of ToS being accepted or Shareaholic.com being reachable.

Reply:
Done.